ui/tests/acceptance/auth/auth-list-test.js

```diff
@@ -80,7 +80,7 @@ module('Acceptance | auth backend list', function (hooks) {
     });
 
     // Test all auth methods, not just those you can log in with
-    methods()
+    filterEnginesByMountType('auth')
```

Comment:
The default for includeEnteprise is false, but I like to call it out explicitly because it indicates to the reader what exactly we're returning
```suggestion
    filterEnginesByMountType({ mountType = 'auth', includeEnterprise = false })
```